src/reportSources/legacyFflogs/eventAdapter/translate.ts

```diff
@@ -242,6 +244,7 @@ export class TranslateAdapterStep extends AdapterStep {
 
 		// Build the new event
 		const overkill = event.overkill ?? 0
+		const bonusPercent = event.bonusPercent || 0
```

Comment:
```suggestion
		const bonusPercent = event.bonusPercent ?? 0
```
We prefer ?? for nullish coalescence, rather than relying on falsey behavior.